vms/platformvm/state/state.go

```diff
@@ -2230,6 +2571,16 @@ func (s *state) writeValidatorDiffs(height uint64) error {
 	return nil
 }
 
+func getOrDefault[K comparable, V any](m map[K]*V, k K) *V {
```

Comment:
A comment explaining that the default value is written to the map would be helpful. Or consider renaming to `getOrAddDefault`

Reply:
did both